akka-remote-tests/src/multi-jvm/scala/akka/remote/TransportFailSpec.scala

```diff
@@ -102,8 +102,9 @@ abstract class TransportFailSpec extends RemotingMultiNodeSpec(TransportFailConf
   override def initialParticipants = roles.size
 
   def identify(role: RoleName, actorName: String): ActorRef = {
-    system.actorSelection(node(role) / "user" / actorName) ! Identify(actorName)
-    expectMsgType[ActorIdentity].ref.get
+    val p = TestProbe()
+    (system.actorSelection(node(role) / "user" / actorName)).tell(Identify(actorName), p.ref)
+    p.expectMsgType[ActorIdentity](remainingOrDefault).ref.get
```

Comment:
Why does this work better than the implicit sender?

Reply:
yes, `identify` was called from a `awaitAssert` and then there is a risk that there are more in flight because of the retries